### DISCUSSION TITLE
How can I create a cell type with multiple buttons inside?

### DISCUSSION THREAD

**Discussion by lucaseduoli** [glideapps/glide-data-grid#613] [General]
I'm using this table for a project, and it requires that one row has two or more buttons on each cell. How can I implement it, considering that each cell is rendered using Canvas?

**Comment by jassmith** [comment]
The dumbest method is to use two columns and the `verticalBorder` to make 2 button cells look like 1 :)

**Comment by lucaseduoli** [comment]
in that case, how can I make the columns header appear as one too? I liked the idea of using verticalBorder to make 2 cells look like one, but the header appears different still.

**Comment by lucaseduoli** [comment]
and one more thing, how can I make the columns move together? They stay together after it has been moved, because of my function that handles the moving, but when I drag the column, only one column follows my mouse pointer.

**Comment by jassmith** [comment]
Oh you, finding all the flaws with my poorly laid plans... I will have to write the multi-button sample eventually I guess. In the mean time I would say continue with what you have and I will get the sample out soon. It is little more than a modification to the links cell which already can do what you want but renders as links instead of buttons.

**Reply by lucaseduoli** [reply]
Hello, I have implemented the code for the multi-buttons... If you want it, I can provide the code for you. It aligns the buttons on the middle and show a "more" button if it cannot fit every button on the cell.

**Reply by jassmith** [reply]
Please!

**Comment by intensions** [comment]
Hey all, I have a similar use case where a single cell contains a list of links. More specifically, thinking it'll be a drilldown cell with an svg icon, and clicking it navigates to a page. Perhaps the fastest way would be to extend the existing drilldown renderer, or a custom provideEditor prop?

Did you end up with a working solution?
